### PR TITLE
Change documentation theme

### PR DESCRIPTION
**Pull request by LeonWubben**
Theme is now sphinx_book_theme in line with other IQM documentations

Changed the logo to transparent

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -95,10 +95,12 @@
 
 # -- Options for HTML output -------------------------------------------------
 
+import sphinx_book_theme
+
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'nature'
+html_theme = 'sphinx_book_theme'
```
Comment (hay-k) [inline docs/conf.py:104]:
```suggestion

html_theme_options = {
    'logo_only': True,
}
```

File: docs/conf.py [hunk 2]
```diff
@@ -112,7 +114,7 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = '_static/images/logo.jpg'
+html_logo = '_static/images/logo.png'
```
Comment (hay-k) [inline docs/conf.py:118]:
```suggestion

html_theme_options = {
    'logo_only': True,
}
```

**Review by hay-k — APPROVED** [review]

File: setup.cfg [hunk 3]
```diff
@@ -53,6 +53,7 @@ dev =
     tox == 3.24.5
 docs =
     sphinx == 4.4.0
+    sphinx-book-theme == 0.1.10
```
Comment (smite) [inline setup.cfg:56]:
```suggestion
    sphinx == 4.5.0
    sphinx-book-theme == 0.3.2
```

Comment (smite) [reply setup.cfg:56]: I would just use the latest versions.

**Review by smite — CHANGES_REQUESTED** [review]

**Review by smite — APPROVED** [review]

**Pull request merged**